Application/EdFi.Security.DataAccess/Repositories/ISecurityRepository.cs

```diff
@@ -4,35 +4,39 @@
 // See the LICENSE and NOTICES files in the project root for more information.
 
 using System.Collections.Generic;
+using Autofac.Extras.DynamicProxy;
+using Castle.DynamicProxy;
 using EdFi.Security.DataAccess.Models;
 
-namespace EdFi.Security.DataAccess.Repositories
-{
-    public interface ISecurityRepository
-    {
-        Action GetActionByHttpVerb(string httpVerb);
+namespace EdFi.Security.DataAccess.Repositories;
 
-        Action GetActionByName(string actionName);
```

Comment:
Please restore `GetActionByHttpVerb`. If we change this interface we will break other code that depends on it.